examples/core/core_2d_camera.py

```diff
@@ -1,10 +1,12 @@
 """
 
-raylib [core] example - 2d camera
+raylib [core] example - 2D Camera
 
 """
-import pyray
 
+# Import
+# ------------------------------------------------------------------------------------
```

Comment:
Ugh, that looks like too much. Everybody knows where imports are in python module.